draft-ietf-quic-transport.md

```diff
@@ -788,6 +788,7 @@ explained in more detail as they are referenced later in the document.
 | 0x04             |  WINDOW_UPDATE     | {{frame-window-update}}    |
 | 0x05             |  BLOCKED           | {{frame-blocked}}          |
 | 0x07             |  PING              | {{frame-ping}}             |
+| 0x08             |  LIMIT_UPDATE      | {{frame-limit-update}}     |
```

Comment:
I really don't like this name.  (I don't like WINDOW_UPDATE either).  How about `MAX_STREAM_ID`.